### PR TITLE
Add reactions preview to search service

### PR DESCRIPTION
**Pull request by breathingdust**
Currently the search service does not include the reactions preview media type header so issue searches do not include the reaction counts this preview provides. The issue struct already includes this [attribute](https://github.com/google/go-github/blob/b06cfbb1abc090e2ff47cac2f0d2d1750f847293/github/issues.go#L52) so adding this preview header is enough to populate it.

This PR adds that preview so this structure is populated.

### CODE REVIEW

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @breathingdust !

Could you please add two new test methods in the `github/search_test.go` file to cover the new case?

* `TestSearchService_Issues`
* `TestSearchService_Issues_coverage`

**Review by gmlewis — APPROVED** [review]
Oh, never mind... I see we have these already.  Thanks.

LGTM.
Awaiting second LGTM before merging.

**Review by wesleimp — APPROVED** [review]
💙

**Comment by gmlewis** [comment]
Thank you, @wesleimp !
Merging.

**Pull request merged**
